packages/block-library/src/block/edit.js

```diff
@@ -117,52 +130,46 @@ function applyInitialOverrides( blocks, overrides = {}, defaultValues ) {
 	} );
 }
 
-function getOverridesFromBlocks( blocks, defaultValues ) {
+function getContentValuesFromInnerBlocks( blocks, defaultValues ) {
 	/** @type {Record<string, Record<string, unknown>>} */
```

Comment:
Nit: The type will then be...

```suggestion
	/** @type {Record<string, { values: Record<string, unknown>> }} */
```